### ISSUE TITLE
[Feature] Steam card farming

### ISSUE THREAD

**Issue by RedSparr0w** [ZixeSea/SteamIdler#8]
Add an idling option for steam card farming

Fall back to dynamic/static idling when not enough games to idle for cards?

**Comment by ZixeSea** [comment]
I don't think `steam-user` as any code that would let you view badge related information outside of badge level. So I don't see a way to get farmable card through `steam-user`. So probably need to find another way to get this info

**Comment by RedSparr0w** [comment]
Yeah, this can be done through https://github.com/DoctorMcKay/node-steamcommunity

Need to fetch the badges pages
```js
community.httpRequest(`https://steamcommunity.com/my/badges/?p=${page}`
```
Then need to parse the HTML to find games with drops remaining and the AppID and pages remaining
Also need to check if newly purchased (don't want to use up the 2 hours playtime if they may want to refund it)
